src/libasr/codegen/asr_to_c.cpp

```diff
@@ -677,6 +682,7 @@ R"(
             exit(1);                                                           \
         }                                                                      \
     }
+#endif
```

Comment:
I think we should move this into `lfortran_intrinsics.h` that we require anyway.